### PR TITLE
Readme clarity

### LINKED ISSUE #73: Clarity of readme
Readme could be better optimised by:
- concatenating commands
- mentioning `npm install`

### PR DESCRIPTION
**Pull request by Nuthi-Sriram**
Fixes https://github.com/litmuschaos/litmus-docs/issues/73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ Additional details on the Docusaurus project can be found [here](https://docusau
 ### Install Node.js
 
 ```bash
-sudo apt-get install python-software-properties
-curl -sL https://deb.nodesource.com/setup_9.x | sudo -E bash -
+sudo apt-get install python-software-properties && curl -sL https://deb.nodesource.com/setup_9.x | sudo -E bash -
```
Comment (chandankumar4) [inline README.md]: This line change is not required

File: README.md [hunk 2]
```diff
@@ -25,16 +24,15 @@ npm install -g yarn
 ### Clone litmus-docs repository
 
 ```bash
-git clone https://github.com/litmuschaos/litmus-docs.git
-cd litmus-docs
+git clone https://github.com/litmuschaos/litmus-docs.git && cd litmus-docs
```
Comment (chandankumar4) [inline README.md]: This line change is not required

File: README.md [hunk 3]
```diff
@@ -25,16 +24,15 @@ npm install -g yarn
...
 ```
 
 ### Start the server
 
 ```bash
-cd litmus-docs/website
-npm start
+cd litmus-docs/website && npm install && npm start
```
Comment (chandankumar4) [inline README.md]: Can you split this in multiple lines

File: website/versioned_docs/version-0.7.0/getstarted.md [hunk 4]
```diff
@@ -209,7 +209,7 @@ kubectl delete ns litmus
 
 ## Example
 
-See [the tutorial](example.html) on running a `pod-delete` chaos experiment on `nginx` application.
+See the [tutorial](https://docs.litmuschaos.io/docs/example/) on running a `pod-delete` chaos experiment on `nginx` application.
```
Comment (chandankumar4) [inline website/versioned_docs/version-0.7.0/getstarted.md]: This line change is not required

Comment (Nuthi-Sriram) [reply website/versioned_docs/version-0.7.0/getstarted.md]: It was throwing a 404 error

Comment (kmjayadeep) [reply website/versioned_docs/version-0.7.0/getstarted.md]: it should be `example.md` instead

File: README.md (hunk 2 above)
Comment (anupriya0703) [reply README.md]: This is not required.

**Comment by chandankumar4** [comment]
Thanks, @Nuthi-Sriram for your contribution, but these changes are not required

**Pull request closed without merging**